File: II sem/Programowanie Obiektowe/Projekty/projekt1/POproject1/Mlecz.cpp
#include "Mlecz.h"

Mlecz::Mlecz(int x, int y) : Roslina(EMOJI ? u8"Ml" : "Ml", 0, x, y)
{

}

void Mlecz::akcja()
{
	for (int i = 0; i < 3; i++)
		Roslina::akcja();
}

Organizm* Mlecz::potomek()
{
	return new Mlecz(*this);
}

File: II sem/Programowanie Obiektowe/Projekty/projekt2/po3/src/Mlecz.java
public class Mlecz extends Roslina {
    public Mlecz(String ikonka, int x_pos, int y_pos, int sila, int inicjatywa, Swiat swiat)
    {
        super(ikonka, x_pos, y_pos, sila, inicjatywa, swiat);
    }

    public Mlecz(Organizm organizm)
    {
        super(organizm);
    }

    @Override
    void kolizja(Organizm organizm)
    {

    }

    @Override
    public void rozmnazanie()
    {
        super.rozmnazanie();
        super.rozmnazanie();
        super.rozmnazanie();
    }
    @Override
    public void moc(Organizm organizm)
    {

    }


    @Override
    Organizm kopiuj(Organizm organizm)
    {
        return new Trawa(organizm);
    }
}
